packages/logging-winston/src/index.js

```diff
@@ -156,9 +156,20 @@ LoggingWinston.prototype.log = function(levelName, msg, metadata, callback) {
 
   var labels = {};
 
-  // Logging proto requires that the label values be strings, so we convert
-  // using util.inspect.
-  if (is.object(metadata)) {
+  // For error messages at severity 'error' and higher, Stackdriver
+  // Error Reporting will pick up error messages if the full stack trace is
+  // included in the textPayload or the message property of the jsonPayload.
+  // https://cloud.google.com/error-reporting/docs/formatting-error-messages
+  //
+  if (is.error(metadata)) {
+    if (msg.length === 0) {
+      msg = metadata.stack;
```

Comment:
Ok, re-reading it now I realize `msg` is not a field but a variable name. Sounds good.